src/constraints.jl

```diff
@@ -727,3 +727,313 @@ function list_of_constraint_types(model::Model)
     return Tuple{DataType, DataType}[(jump_function_type(model, f), s)
                                      for (f,s) in list]
 end
+
+
```

Comment:
I'd recommend moving this code into a new file called `lp_sensitivity.jl`. It's pretty specialized and doesn't have much to do with the rest of constraints.jl.